service/src/main/kotlin/no/nav/su/se/bakover/service/skatt/JournalførSkattDokumentService.kt

```diff
@@ -0,0 +1,68 @@
+package no.nav.su.se.bakover.service.skatt
+
+import arrow.core.Either
+import arrow.core.getOrElse
+import arrow.core.left
+import no.nav.su.se.bakover.client.dokarkiv.DokArkiv
+import no.nav.su.se.bakover.client.dokarkiv.Journalpost
+import no.nav.su.se.bakover.client.dokarkiv.JournalpostSkatt.Companion.lagJournalpost
+import no.nav.su.se.bakover.common.journal.JournalpostId
+import no.nav.su.se.bakover.domain.brev.KunneIkkeJournalføreDokument
+import no.nav.su.se.bakover.domain.person.PersonService
+import no.nav.su.se.bakover.domain.sak.SakService
+import no.nav.su.se.bakover.domain.skatt.DokumentSkattRepo
+import no.nav.su.se.bakover.domain.skatt.Skattedokument
+import no.nav.su.se.bakover.service.journalføring.JournalføringOgDistribueringsResultat
+import no.nav.su.se.bakover.service.journalføring.JournalføringOgDistribueringsResultat.Companion.logResultat
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+/**
+ * journalfører dokumenter tilhørende skatt. Ment å bli kallt fra en jobb
+ */
+class JournalførSkattDokumentService(
+    private val dokArkiv: DokArkiv,
+    private val sakService: SakService,
+    private val personService: PersonService,
+    private val dokumentSkattRepo: DokumentSkattRepo,
+) {
+    val log: Logger = LoggerFactory.getLogger(this::class.java)
+
+    fun journalfør() {
```

Comment:
Hvordan tester du denne? Sjekker basen? Ref tilsvarende metoder i journalførDok og DistDok som har åpnet private funs som internal.

Kunne man gjort det likt for disse 3?